### PR TITLE
Print full data to log

### PR DESCRIPTION
**Pull request by jonmv**
@bjorncs please review and merge. We're hiding results now, after `execute_bg`.

### CODE REVIEW

**Review by bjorncs — APPROVED** [review]

File: tests/performance/programmatic_feed_client/java-feed-client/pom.xml [hunk 1]
```diff
@@ -13,6 +13,10 @@
     <relativePath>../../../pom.xml</relativePath>
   </parent>
 
+  <properties>
```
Comment (bjorncs) [inline tests/performance/programmatic_feed_client/java-feed-client/pom.xml:16]: It's given from the parent pom afaik

File: tests/performance/programmatic_feed_client/programmatic_feed_client.rb [hunk 2]
```diff
@@ -130,7 +145,7 @@ def run_benchmark_program(container_node, vespa_route, main_class, output, text)
     java_cmd =
       "java #{perfmap_agent_jvmarg} -cp #{java_client_src_root}/target/java-feed-client-1.0.jar " +
         "-Dvespa.test.feed.route=#{vespa_route} " +
-        "-Dvespa.test.feed.documents=#{DOCUMENTS} " +
+        "-Dvespa.test.feed.documents=#{(DOCUMENTS / (size / 10) ** 0.5).to_i} " +
```
Comment (bjorncs) [inline tests/performance/programmatic_feed_client/programmatic_feed_client.rb:148]: 🤯

**Review by bjorncs — APPROVED** [review]

File: tests/performance/programmatic_feed_client/java-feed-client/pom.xml (hunk 1 above)
Comment (jonmv) [reply tests/performance/programmatic_feed_client/java-feed-client/pom.xml:16]: Not `vespa.version`?

File: tests/performance/programmatic_feed_client/programmatic_feed_client.rb (hunk 2 above)
Comment (jonmv) [reply tests/performance/programmatic_feed_client/programmatic_feed_client.rb:148]: ƪ(`▿▿▿▿´ƪ)

File: tests/performance/programmatic_feed_client/java-feed-client/pom.xml (hunk 1 above)
Comment (bjorncs) [reply tests/performance/programmatic_feed_client/java-feed-client/pom.xml:16]: Ok, it should be added there then.

**Pull request merged**